fastlane/Fastfile

```diff
@@ -166,18 +166,33 @@ def push_pods
   pod_push(path: "PurchasesHybridCommon.podspec", synchronous: true)
 end
 
-def carthage_archive
+def archive
+  framework_filename = "PurchasesHybridCommon.framework"
+  build_app(
+    workspace: "ios/PurchasesHybridCommon/PurchasesHybridCommon.xcworkspace",
+    configuration: "Release",
+    scheme: "PurchasesHybridCommon",
+    silent: true,
+    clean: true,
+    derived_data_path: "derived_data/",
+    output_name: framework_filename,
+    destination: 'generic/platform=iOS'
+  )
```

Comment:
this is also faster, since it's only building the actual framework we want, in the actual platform we want.